backend/src/main/kotlin/hu/bme/sch/cmsch/component/ComponentApiBase.kt

```diff
@@ -127,7 +127,7 @@ abstract class ComponentApiBase(
     }
 
     open fun onInit() {
-        // The compoennt is loaded
+        // The component is loaded
     }
```

Comment:
Please include the invalidation of BanList when someone changes the RiddleComponent values. Make sure it is triggered when the cache invalidation comes from /remote-api/riddle/reload-component-config

Reply:
done